File: tkpdtopadssdk/src/main/java/com/tokopedia/topads/sdk/view/adapter/viewholder/LoadingViewHolder.java
package com.tokopedia.topads.sdk.view.adapter.viewholder;

import android.support.annotation.LayoutRes;
import android.view.View;

import com.tokopedia.topads.sdk.R;
import com.tokopedia.topads.sdk.base.adapter.viewholder.AbstractViewHolder;
import com.tokopedia.topads.sdk.view.adapter.viewmodel.discovery.LoadingViewModel;

/**
 * @author by errysuprayogi on 4/18/17.
 */

public class LoadingViewHolder extends AbstractViewHolder<LoadingViewModel> {

    @LayoutRes
    public static final int LAYOUT = R.layout.layout_loading;

    public LoadingViewHolder(View itemView) {
        super(itemView);
    }

    @Override
    public void bind(LoadingViewModel element) {

    }
}

File: features/transaction/express_checkout/src/main/java/com/tokopedia/expresscheckout/view/variant/viewholder/LoadingViewHolder.kt
package com.tokopedia.expresscheckout.view.variant.viewholder

import android.view.View
import com.tokopedia.abstraction.base.view.adapter.viewholders.LoadingViewholder
import com.tokopedia.expresscheckout.R

/**
 * Created by Irfan Khoirul on 30/01/19.
 */

class LoadingViewHolder(val view: View) : LoadingViewholder(view) {

    companion object {
        val LAYOUT = R.layout.item_loading_shimmering_atc_express
    }

}